### ISSUE TITLE
[Feature Request] "Tree Variants" tweaks

### ISSUE THREAD

**Issue by vico93** [Rebirth-of-the-Night/Quark#25]
- Change the Blossoming tree (log, leaves and sapling) to match 1.20's Cherry tree textures and sound;
- If BoP is ON, automatically disable Blossoming trees from worldgen (while keeping Swamp Oak Trees intact).

**Comment by Foreck1** [comment]
I would like to add that while both look like cherry trees, both have different names, slightly different textures, and Quark's spawn exclusively on extreme hills (also both have separate dynamic trees addons). So the second suggestion should be an option, probably off by default.

**Comment by vico93** [comment]
Eh, you have a point. But considering that there is no Cherry Grove biome on vanilla 1.12, spawning on Extreme Hills is the least of the issues.

**Comment by Foreck1** [comment]
My point was that, were you to install BoP, Quark's cherry trees wouldn't overlap with them as they would generate in totally different biomes for totally different purposes. If you just install BoP, for example, you won't get the "sakura hills" that Quark intended to add, and if you just install Quark, you won't get the "cherry blossom grove" that BoP intended to add.

**Comment by vico93** [comment]
Understand. So the "reskin" request should do more sense for BoP's cherry trees.

**Comment by Foreck1** [comment]
Not necessarily, the reskin is fine (first suggestion); but deactivating Quark's cherry trees when BoP is installed should be off by default (second suggestion).

**Comment by vico93** [comment]
ahh i understand, sorry for my confusion
